### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by flo-butterfly** [nocodb/nocodb#7484] [Q&A] [answered]
On your website you mention the product is free during the technical preview.

I was wondering what kind of pricing you plan for self hosted implementation. Will it remain free?

Thank you

**Comment by mertmit** [comment, marked as answer]
Hi @flo-butterfly, thank you for asking.
NocoDB is an open-source project and **it is freely available for self-hosting and it will stay that way**.
Our cloud & enterprise offerings are for those who prefer a hosted solution instead of a self-hosted one or needs enterprise services from us, and we plan to stick with a starter plan for cloud as well.

**Reply by AdsonCicilioti** [reply]
Hi @mertmit !

Does the self-hosted have or will there be any limitations in relation to the Cloud versions, and if so, what are they?

**Reply by skyemk** [reply]
They got the infamous SSO tax

**Reply by surfmuggle** [reply]
Quote from [Feature: Support HTTP Header Based Authentication](https://github.com/nocodb/nocodb/issues/9818):

> After reviewing issues https://github.com/nocodb/nocodb/issues/517, https://github.com/nocodb/nocodb/issues/742, and https://github.com/nocodb/nocodb/pull/3979, I understand that NocoDB has made the strategic decision to keep SSO features in the enterprise edition. This decision helps maintain the project's commercial viability. 

The linked Feature request proposes adding support for HTTP Header Based Authentication - allowing NocoDB to read and validate a predefined authentication header for user identification. Third party

**Reply by mertmit** [reply]
See following for clear comparison:
https://docs.nocodb.com/cloud-enterprise-edition/community-vs-paid-editions/

@surfmuggle we will discuss internally and keep updates on that issue
